workflow/client.go

```diff
@@ -234,15 +234,17 @@ func (c *client) CompleteSuccessfulActivity(workflowID, activityID string, resul
 	return response.Payload, nil
 }
 
-func (c *client) CompleteCancelledActivity(workflowID, activityID, details string) (*models.Activity, error) {
+// CompleteCancelledActivity will sent an activity with a cancelled status to the workflow API.  workflowID, activityID,
+// and reason are required.
+func (c *client) CompleteCancelledActivity(workflowID, activityID, reason, details string) (*models.Activity, error) {
 	token, err := c.tokenFetcher.Token(c.audience)
 	if err != nil {
 		return nil, err
 	}
 	cancelledActivity := &models.Activity{
 		ID:     swag.String(activityID),
 		Status: swag.String(models.ActivityStatusCancelled),
-		Error:  &models.ActivityError{Details: details},
+		Error:  &models.ActivityError{Reason: swag.String(reason), Details: details},
```

Comment:
Formatting? Maybe Details on another line to be consistent.

Reply:
Ah, I personally only break initializers across lines if the line is too long.  1 or 2 fields, is okay.  More than that and probably want multiple lines.